test/server.cpp

```diff
@@ -369,7 +368,7 @@ static int deleteRequest(struct mg_connection* conn) {
     auto has_json_header = false;
     for (int i = 0; i < num_headers; ++i) {
         auto name = headers[i].name;
-        if (std::string{"Content-Type"} == headers[i].name &&
+        if (std::string{"Content-Type"} == name &&
```

Comment:
Hmm, it looks like `name` isn't used anywhere but here. Let's just replace it with `headers[i].name` and remove the above variable definition.